src/services/goals.js

```diff
@@ -568,6 +568,7 @@ export async function goalsByIdsAndActivityReport(id, activityReportId) {
         include: [
           {
             model: ObjectiveResource,
+            separate: true,
```

Comment:
Adding separate seem to give it a 50% speed increase. However I'm skeptical of how much we will actually see.